### PR TITLE
Add exportloopref linter

### PR DESCRIPTION
**Pull request by sayboras**
Fixes https://github.com/golangci/golangci-lint/issues/1041

### CODE REVIEW

File: pkg/lint/lintersdb/manager.go [hunk 1]
```diff
@@ -272,6 +272,9 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 		linter.NewConfig(golinters.NewNestif()).
 			WithPresets(linter.PresetComplexity).
 			WithURL("https://github.com/nakabonne/nestif"),
+		linter.NewConfig(golinters.NewExportLoopRef()).
+			WithPresets(linter.PresetStyle).
```
Comment (jirfag) [inline pkg/lint/lintersdb/manager.go:276]: isn't it `bugs` preset?

File: pkg/lint/lintersdb/manager.go [hunk 2]
```diff
@@ -272,6 +272,9 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
...
+			WithURL("github.com/kyoh86/exportloopref"),
```
Comment (ernado) [inline pkg/lint/lintersdb/manager.go:277]: Please add `https://`

Comment (sayboras) [reply pkg/lint/lintersdb/manager.go:277]: Thanks :100:

File: pkg/lint/lintersdb/manager.go (hunk 1 above)
Comment (sayboras) [reply pkg/lint/lintersdb/manager.go]: you are right, bugs present is more appropriate :100:

**Comment by kyoh86** [comment]
`looppointer` can be an option.
Maybe this is not so important, but I'd like to mention it.

Since `exportloopref` cannot find some diagnostics like below.

Case 1: pass the pointer to the function to export.

Case 2: pass the pointer to the local variable, and export it.

```go
package main

type List []*int

func (l *List) AppendP(p *int) {
	*l = append(*l, p)
}

func main() {
	var slice []*int
	list := List{}

	println("loop expect exporting 10, 11, 12, 13")
	for _, v := range []int{10, 11, 12, 13} {
		list.AppendP(&v) // Case 1: wanted "exporting a pointer for the loop variable v", but cannot be found

		p := &v                  // p is the local variable
		slice = append(slice, p) // Case 2: wanted "exporting a pointer for the loop variable v", but cannot be found
	}

	println(`slice expecting "10, 11, 12, 13" but "13, 13, 13, 13"`)
	for _, p := range slice {
		printp(p)
	}
	println(`array expecting "10, 11, 12, 13" but "13, 13, 13, 13"`)
	for _, p := range ([]*int)(list) {
		printp(p)
	}
}

func printp(p *int) {
	println(*p)
}
```

So we may use `looppointer` instead of `exportloopref` if we want to find them all (with some false-positives).

https://github.com/kyoh86/exportloopref#known-false-negatives

**Pull request merged**